components/board/fake/board.go

```diff
@@ -461,40 +461,40 @@ func (h *I2CHandle) Close() error {
 	return nil
 }
 
-// A Analog reads back the same set value.
-type Analog struct {
+// An AnalogReader reads back the same set value.
+type AnalogReader struct {
+	pin        string
 	Value      int
 	CloseCount int
 	Mu         sync.RWMutex
-	chipSelect string
```

Comment:
You seem to be replacing the chipSelect with the pin. I had thought the chipSelect told the board how to talk to the MCP300x over SPI, and the pin was which reader within the MCP300x to talk to. If you remove the chipSelect like this, how does the board talk to it at all?

Reply:
Since this is fake board I don't think it should have the MCP300x chip anymore since its not part of the common board config.